update_image.pl

```diff
@@ -404,7 +426,7 @@ sub processConvergedSections {
 }
 else
 {
-    # Inject ECC into HBD (hostboot targeting) output binary
+    # Inject ECC into HBD_RO (hostboot targeting) output binary
```

Comment:
HBD_RO is kinda meaningless for p8 branch right .. I don't think any of this gets executed;  IF it did, the mods above in the create_pnor_image.pl would not find a p8 RW partitions in this branch.

Reply:
Correct.